src/renderer/screens/platform/TwitterBanner.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React, { useMemo } from "react";
+import styled from "styled-components";
+import Box from "~/renderer/components/Box/Box";
+import Alert from "~/renderer/components/Alert";
+import Text from "~/renderer/components/Text";
+import { useTranslation } from "react-i18next";
+
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+const Container: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  mb: 24,
+}))``;
+
+const twitterHashtag = "LedgerLiveApp";
+
+export default function TwitterBanner() {
+  const { t } = useTranslation();
+
+  const url = useMemo(() => {
+    const urlObj = new URL("https://twitter.com/intent/tweet");
```

Comment:
Move this to `config/urls.js` so that it's not hardcoded here

Reply:
Thanks for the feedback, it has been updated here d95d571